main.go

```diff
@@ -103,9 +90,19 @@ func main() {
 		&defaults.GKENodePoolProfile{},
 		&objectstore.ManagedAmazonBucket{},
 		&objectstore.ManagedGoogleBucket{},
-		&route53model.Route53Domain{},
-	).Error; err != nil {
+		&route53model.Route53Domain{}}
+
+	var tableNames string
+	for _, table := range tables {
+		tableNames += fmt.Sprintf(" %s", db.NewScope(table).TableName())
+	}
+
+	logger.WithFields(logrus.Fields{
+		"table_names": tableNames,
```

Comment:
`strings.TrimSpace` would remove the leading space in the field

Reply:
fixed, thx